### ISSUE TITLE
v2.0.1 requires a variable named map

### ISSUE THREAD

**Issue by CraigVA** [Esri/esri-leaflet#814]
Using Leaflet 1.00 rc2 and Esri-Leaflet 2.0.1, I made a map like this where the map variable is named m:

```
var m = L.map('map').setView([37.75,-122.45], 12);
L.esri.basemapLayer("Topographic").addTo(m);
```

In Chrome and IE 11, this throws an uncaught reference error, and it gets thrown again each time the map is zoomed or panned, until eventually it's frozen.  (in Firefox it doesn't throw a reference error if you have a div named map, but will if you make a version with the div named something else.)

It's cause by line 166 of Util.js, where it expects map to already be defined:
```
export function _updateMapAttribution (evt) {
  map = evt.target;
```

And I'm not an expert, but it seems like redefining the map object every time the map attribution gets updated is a bad idea, even when you do have the right reference.

**Comment by jgravois** [comment]
thanks for reporting.  i proposed a fix in #809 and plan to tag `2.0.2` shortly after it's merged.

**Comment by CraigVA** [comment]
Glad to see it's already being addressed, I didn't think to also check the pull requests before posting.  I initialized and renamed the variable in my local copy and it's working fine now.

**Comment by jgravois** [comment]
fix has been packaged in [`2.0.2`](https://github.com/Esri/esri-leaflet/releases/tag/v2.0.2)
